app/controlplane/internal/dispatcher/dispatcher.go

```diff
@@ -101,11 +102,28 @@ func (d *FanOutDispatcher) Run(ctx context.Context, opts *RunOpts) error {
 		return fmt.Errorf("workflow not found")
 	}
 
+	wfRun, err := d.wfRunUC.View(ctx, opts.OrgID, opts.WorkflowRunID)
+	if err != nil {
+		return fmt.Errorf("finding workflow: %w", err)
+	} else if wfRun == nil {
+		return fmt.Errorf("workflowRun not found")
+	}
+
 	workflowMetadata := &sdk.ChainloopMetadata{
```

Comment:
I needed to increase the amount of metadata that we injected in the plugin. Including now information about the workflow run